Formula/emacs-plus@28.rb

```diff
@@ -10,6 +10,7 @@ class EmacsPlusAT28 < EmacsBase
 
   # Opt-out
   option "without-cocoa", "Build a non-Cocoa version of Emacs"
+  option "without-modern-fonts", "Disable modern font rendering (not recommended)"
```

Comment:
What is disadvantage of `modern-fornts`? I mean, is it really should be an opt-out option or just an unconditional feature?

Reply:
None that I'm aware of, but it's still considered new enough that I think it's worth having the option to turn it off.